### PR TITLE
more halloween costumes

### PR DESCRIPTION
**Pull request by MadmanMartian**
Adds 3 halloween costume pieces, entire thing courtesy of @Blithering 

- Neck bolts. For that frankenstein look
- Alien antennae. Don't worry, not the reddit one.
- Glowy fangs. Absorbs light from the vicinity, and glows when you're hiding in maintenance ready to spook the head of security while they're hunting for clowns to kill ~~before subsequently being beaten to death by the head of security while being holy waterboarded by the chaplain~~

All 3 give alternate costumes to Ian the Corgi

:cl:
* rscadd: More variety added to the halloween costume crate

### CODE REVIEW

**Comment by CptWad** [comment]
I DEMAND PHOTOS

**Comment by SonixApache** [comment]
post pictures

**Comment by Blithering** [comment]
halloweencorgi

File: code/modules/mob/living/simple_animal/friendly/corgi.dm [hunk 1]
```diff
@@ -395,6 +395,24 @@
 			SetLuminosity(1)
 			valid = 1
 		*/
+		if(/obj/item/clothing/head/alien_antenna)
+			name = "Al-Ian"
```
Comment (Aurx) [inline code/modules/mob/living/simple_animal/friendly/corgi.dm:399]: >AI-Ian
>Not A-Ian

ONE FUCKING JOB

Comment (Exxion) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:399]: I don't get it

**Review by PJB3005 — APPROVED** [review]

File: code/modules/mob/living/simple_animal/friendly/corgi.dm (hunk 1 above)
Comment (MadmanMartian) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:399]: It's AL-IAN though

Comment (SonixApache) [reply code/modules/mob/living/simple_animal/friendly/corgi.dm:399]: It's not the same A sound, but you kinda lose the point if you make it a-ian

**Pull request merged**